src/Assets/Javascript/map.js

```diff
@@ -59,47 +64,70 @@ const initGoogleMaps = () => {
   const infoWindow = new google.maps.InfoWindow()
   const markers = []
 
-  google.maps.event.addListener(map, 'click', function () {
+  google.maps.event.addListener(map, "click", function() {
     infoWindow.close()
   })
 
   for (let i = 0, length = locations.length; i < length; i++) {
     const data = locations[i]
     const latLng = new google.maps.LatLng(data.lat, data.lng)
 
-    const marker = new google.maps.Marker({
+    const marker = new MarkerWithLabel({
+      map: map,
       position: latLng,
-      map,
-      title: data.title,
       icon: locationMarker,
+      title: data.title,
+      labelContent: data.title,
+      labelAnchor: new google.maps.Point(-20, 25),
+      labelClass: "my-custom-class-for-label", // the CSS class for the label
+      labelInBackground: true,
       zIndex: 2
-    });
+    })
 
-    (((marker, data) => {
-      google.maps.event.addListener(marker, 'click', e => {
+    // const marker = new google.maps.Marker({
+    //   position: latLng,
+    //   map,
+    //   title: data.title,
+    //   icon: locationMarker,
+    //   zIndex: 2
+    // })
+    ;((marker, data) => {
```

Comment:
semi colon?

Reply:
Yeah; it's an auto formatting thing. If you want to write an [IIFE](https://en.wikipedia.org/wiki/Immediately-invoked_function_expression), you need a semicolon either at the end of the previous statement or at the start of the IIFE. It's more defensive to write it at the start because it can't break suddenly when you make changes to other lines, which is less surprising.